### PR TITLE
Fix random seed. Remove obsolete try/except.

### PR DESCRIPTION
**Pull request by AlexOlza**

### CODE REVIEW

File: main/cluster/predict.py [hunk 1]
```diff
@@ -26,19 +26,15 @@ def configure(configname=None):
...
                 # print (tb.format_exc())
                 print (exc)
-    # print(configuration)
     conf=Struct(**configuration)
     conf.TRACEBACK=True
     conf.VERBOSE=True
     if not config.configured:
         config.configure(conf) # configure() receives a python module
     assert config.configured
-    # for statement in config.IMPORTS:  #FIXME is this ugly code?
```
Comment (AlexOlza) [inline main/cluster/predict.py:36]: Remove this. It was there because I thought lacking imports were the origin of issue #8, but it's not the case.

File: main/cluster/predict.py [hunk 2]
```diff
@@ -81,30 +77,23 @@ def predict_save(yr,model,X,y,columns=config.COLUMNS[0]):
...
 import pandas as pd
 import configurations.utility as util
-# from dataManipulation.generarTablasVariables import load
 from dataManipulation.dataPreparation import getData
 from sklearn.metrics import roc_auc_score
 import numpy as np
 from sklearn import linear_model
 from main.SafeLogisticRegression import SafeLogisticRegression
 import importlib
+np.random.seed(config.SEED)
```
Comment (AlexOlza) [inline main/cluster/predict.py:86]: I checked, and this is key for reproducible local predictions, but they are still different to those of the cluster.

File: main/cluster/predict.py [hunk 3]
```diff
@@ -81,30 +77,23 @@ def predict_save(yr,model,X,y,columns=config.COLUMNS[0]):
...
 #%%
 if __name__=='__main__':
         
     # FIXME STRUCT KEYS TO INT, FIX GENERARTABLASVARIABLES.RETRIEVE INDICE
-    configname='/home/aolza/Desktop/estratificacion/configurations/used/randomForest20211217_153917.json'
```
Comment (AlexOlza) [inline main/cluster/predict.py:95]: This should be removed for #23 .

File: main/cluster/predict.py [hunk 4]
```diff
@@ -81,30 +77,23 @@ def predict_save(yr,model,X,y,columns=config.COLUMNS[0]):
...
+    configname='/home/aolza/Desktop/estratificacion/configurations/used/randomForest20211222_161609.json'
     # configuration=configure(configname)
-    modelfilename='/home/aolza/Desktop/estratificacion/models/urgcms_excl_hdia_nbinj/randomForest20211217_153917.joblib'
```
Comment (AlexOlza) [inline main/cluster/predict.py:97]: Removed because  #8 is solved.

File: main/cluster/predict.py (hunk 2 above)
Comment (AlexOlza) [reply main/cluster/predict.py:87]: We also get reproducibility in the cluster, but not between systems.

File: main/cluster/predict.py [hunk 5]
```diff
@@ -67,6 +67,7 @@ def predict_save(yr,model,X,y,columns=config.COLUMNS[0]):
             util.vprint(i,'/',n)
             chunk = X.iloc[index_slice] # your dataframe chunk ready for use
             ychunk=y.iloc[index_slice]
+            seed= np.random.default_rng(42)
```
Comment (AlexOlza) [inline main/cluster/predict.py:70]: This is unnecessary.
